packages/xstate-graph/test/graph.test.ts

```diff
@@ -158,63 +197,111 @@ describe('@xstate/graph', () => {
 
   describe('getShortestPaths()', () => {
     it('should return a mapping of shortest paths to all states', () => {
-      const paths = getShortestPaths(lightMachine) as any;
+      const paths = getShortestPlans(lightMachine) as any;
 
-      expect(paths).toMatchSnapshot('shortest paths');
+      expect(getPathsMapSnapshot(paths)).toMatchSnapshot('shortest paths');
     });
 
     it('should return a mapping of shortest paths to all states (parallel)', () => {
-      const paths = getShortestPaths(parallelMachine) as any;
-      expect(paths).toMatchSnapshot('shortest paths parallel');
+      const paths = getShortestPlans(parallelMachine) as any;
+      expect(getPathsMapSnapshot(paths)).toMatchSnapshot(
+        'shortest paths parallel'
+      );
     });
 
     it('the initial state should have a zero-length path', () => {
+      const shortestPaths = getShortestPlans(lightMachine);
+
       expect(
-        getShortestPaths(lightMachine)[
-          JSON.stringify(lightMachine.initialState.value)
-        ].paths[0].segments
+        shortestPaths.find((plan) =>
+          plan.state.matches(lightMachine.initialState.value)
```

Comment:
looking here - I would have no idea that `plan.state` is holding the "final"/target/end state of the whole "plan". Maybe this could use a more descriptive name?

Reply:
Could you provide a suggestion?

Reply:
We can discuss removing plans in a future PR, since the focus is on `xstate/test`